Classes/BITUpdateManager.m

```diff
@@ -380,19 +379,19 @@ - (instancetype)init {
     _expiryDate = nil;
     _checkInProgress = NO;
     _dataFound = NO;
-    _updateAvailable = NO;
+    self.updateAvailable = NO;
```

Comment:
❓ Why are you using the property accessors here?

Reply:
Thx @lumaxis =)